### DISCUSSION TITLE
Does plasma-manager provide a good way to handle gtk themeing ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [nix-community/plasma-manager#435] [Q&A] [answered]
Hey, everything is in the title.
Although home manager itself provides a way to configure it, it somehow crashes on my install... So before debugging that i was wondering if plasma-manager provided an alternative.

**Comment by HeitorAugustoLN** [comment, marked as answer]
No, generally it is handled with home-manager, I don't know why it is crashing for you. Because that is what most people use for configuring gtk declaratively

**Reply by SpidFightFR** [reply]
> No, generally it is handled with home-manager, I don't know why it is crashing for you. Because that is what most people use for configuring gtk declaratively

Weird indeed... i'll need to check everything again.
